.github/workflows/atlas-image-build.yaml

```diff
@@ -0,0 +1,62 @@
+name: build-mongodb-atlas-tools-image
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'MongoDB Atlas image tag in the format x.x.x'
+        required: true
+        type: string
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: kanisterio/mongodb-atlas
+
+jobs:
+  check-files:
+  runs-on: ubuntu-latest
+    outputs:
+      changed: ${{ steps.changed-files.outputs.any_changed }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: tj-actions/changed-files@v35
```

Comment:
Is this verified GH action? If not, is there a possibility to use a verified action here.

Reply:
Yes, I have verified this when we added kanister-build-image workflow.

Reply:
what I meant is, is this GitHub Action verified in the GitHub Actions marketplace.

Reply:
yes. https://github.com/marketplace/actions/changed-files#usage